starfish/intensity_table/intensity_table.py

```diff
@@ -9,7 +9,12 @@
 
 from starfish.expression_matrix.expression_matrix import ExpressionMatrix
 from starfish.types import Axes, DecodedSpots, Features, LOG, SpotAttributes, STARFISH_EXTRAS_KEY
+from starfish.types._constants import OverlapStrategy
```

Comment:
consider importing `OverlapStrategy` into `starfish.types.__init__`, especially if you foresee users actually using this enum.   If it's going to be for internal use only, this is okay.